ada-project-docs/wave_01.md

```diff
@@ -397,4 +392,4 @@ Status: `200`
 
 ## Tests
 
-There are no Pytest tests for this project.  There are Postman test scripts for all of these endpoints.  If you want to write your own tests for this project, we recommend using the tests in Task List as a template.
\ No newline at end of file
+There are no Pytest tests for this project.  There are Postman test scripts for all of these endpoints.  If you want to write your own tests for this project, we recommend using the tests in Task List as a template.
```

Comment:
I'm guessing this is an unintended change caused by your editor inserting a trailing newline? Don't know how easy it is to prevent that from happening. FWIW, text files _are_ supposed to end with newlines, but they get left off all the time... So I'm pro add newlines!